tgstation.dme

```diff
@@ -4066,5 +4066,6 @@
 #include "modular_sand\code\modules\vending\autodrobe.dm"
 #include "modular_sand\code\modules\vending\clothesmate.dm"
 #include "modular_sand\code\modules\vending\kinkmate.dm"
+#include "modular_sand\icons\obj\machines\cryopod.dm"
```

Comment:
```suggestion
```
I don't even know where did this come from, but just take this down and it should be good to go